packages/patternfly-3/patternfly-react/src/components/DualListSelector/components/DualListItem.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { TypeAheadSelect } from '../../../components/TypeAheadSelect';
+import { noop } from '../../../common/helpers';
+
+const { Highlighter } = TypeAheadSelect;
+
+const DualListItem = ({
+  checked,
+  className,
+  position,
+  parentPosition,
+  value,
+  label,
+  filterTerm,
+  onChange,
+  side,
+  category
+}) => {
+  const cx = classNames('dual-list-pf-item', className, checked ? ' selected' : '');
+  return (
+    <label className={cx}>
+      <input
+        type="checkbox"
+        data-position={position}
+        data-parent-position={parentPosition}
+        onChange={onChange}
+        checked={checked || false}
+        value={value}
+        data-side={side}
+        category={category}
```

Comment:
Yes, I was requested by a member of the UI team to use it instead of regular attribute.
it is used in all event handlers by accessing the `event.target.dateset` object.

Reply:
=) little bit of confusion between both of us, my vague comment started it, we're on the same page now, referring to `category={category}`

Reply:
haha yeah sorry, my bad ! just fixed it now